packages/oasis-actions/src/helpers/ajna/index.ts

```diff
@@ -0,0 +1,106 @@
+import BigNumber from 'bignumber.js'
+import { ethers } from 'ethers'
+
+import poolAbi from '../../../../../abi/external/ajna/ajnaPoolERC20.json'
+import { getAjnaValidations } from '../../strategies/ajna/earn/validations'
+import { AjnaEarnPosition } from '../../types/ajna'
+import { AjnaPool } from '../../types/ajna/AjnaPool'
+import {
+  Address,
+  AjnaDependencies,
+  AjnaEarnActions,
+  AjnaError,
+  AjnaWarning,
+  Strategy,
+} from '../../types/common'
+
+export interface AjnaEarnArgs {
+  poolAddress: Address
+  dpmProxyAddress: Address
+  quoteAmount: BigNumber
+  quoteTokenPrecision: number
+  price: BigNumber
+  position: AjnaEarnPosition
+}
+
+export const prepareAjnaPayload = <T extends { pool: AjnaPool }>({
+  dependencies,
+  args,
+  targetPosition,
+  errors,
+  warnings,
+  data,
+}: {
+  dependencies: AjnaDependencies
+  targetPosition: T
+  errors: AjnaError[]
+  warnings: AjnaWarning[]
+  data: string
+  args: {
+    position: T
+    quoteAmount: BigNumber
+    quoteTokenPrecision: number
+  }
+}): Strategy<T> => {
+  const isDepositingEth =
+    args.position.pool.collateralToken.toLowerCase() === dependencies.WETH.toLowerCase()
+
+  return {
+    simulation: {
+      swaps: [],
+      errors,
+      warnings,
+      targetPosition,
+      position: targetPosition,
+    },
+    tx: {
+      to: dependencies.ajnaProxyActions,
+      data,
+      value: isDepositingEth
+        ? ethers.utils.parseUnits(args.quoteAmount.toString(), args.quoteTokenPrecision).toString()
```

Comment:
What do you think about using `parsEther` instead of `parseUnits`? In here quoteToken can only be ETH.